examples/ibm-iam-accessgroups/main.tf

```diff
@@ -0,0 +1,10 @@
+provider "ibm" {
+}
+
+resource "ibm_iam_access_group" "accgroup" {
```

Comment:
format the example
And include the terraform block to download from registry

Reply:
@hkantare format fixed, but did not get your second comment. can you give some reference to follow?

Reply:
From terraform 0.13 and higher verison to make an example to work we need to add a block to download provider from registry
https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/examples/ibm-watson-query/versions.tf
like this block

Reply:
sure